qml/PingStatus.qml

```diff
@@ -16,7 +16,7 @@ Item {
         color: "black"
         opacity: 0.75
         height: innerCol.height
-        width: innerCol.width*1.2
+        width: parent.width * 0.2
```

Comment:
This is huge. 
![screenshot from 2018-05-28 10-34-27](https://user-images.githubusercontent.com/1215497/40617197-01635ad6-6264-11e8-8787-dedff26f15b7.png)

Reply:
So? it keeps the numbers from all jumping around when you get 2 billion for length or something when there is bugs.

Reply:
> all jumping around when you get 2 billion for length

If the number can go this big, use toExponential(5) or something like it to short the value.

Reply:
if the numbers are changing at 30Hz, decimal will be able to see there is a problem with human eyes.
toExponential maybe not

Reply:
fixed it all lmk if you like it